app/services/subscriptions/dates_service.rb

```diff
@@ -0,0 +1,304 @@
+# frozen_string_literal: true
+
+module Subscriptions
+  class DatesService < BaseService
```

Comment:
Do we need to inherit from `BaseService`?

Reply:
Actually, not at all :+1: